### PR TITLE
Keep the red pause indicator in the events panel from shifting the layout when it changes

### LINKED ISSUE #4362: Red line shouldn't have jumping effect in Events
https://www.loom.com/share/9186855ea0a749619fe7d8ff63314f62

Right now things are jumping around in events because of the red line moving. The line shouldn't push things around. Notice how in console things don't jump.

### PR DESCRIPTION
**Pull request by jaril**
Fix #4362.

https://user-images.githubusercontent.com/15959269/141238834-6fc1e348-abf8-4070-8ad2-852f9b5c2a59.mov

### CODE REVIEW

File: src/ui/components/Events/Events.tsx [hunk 1]
```diff
@@ -9,9 +9,24 @@ const { getExecutionPoint } = require("devtools/client/debugger/src/reducers/pau
...
+function CurrentTimeLine({
+  currentEventIndex,
+  index,
+}: {
+  currentEventIndex: number;
+  index: number;
+}) {
+  return (
+    <div
+      className={classNames("w-full m-0", {
```
Comment (jcmorrow) [inline src/ui/components/Events/Events.tsx:23]: Do you need the `w-full` here? I think the div should be full-width by default, as it's a block-level element.

File: src/ui/components/Events/Events.tsx [hunk 2]
```diff
@@ -9,9 +9,24 @@ const { getExecutionPoint } = require("devtools/client/debugger/src/reducers/pau
...
+        "bg-secondaryAccent": currentEventIndex === index,
```
Comment (jcmorrow) [inline src/ui/components/Events/Events.tsx:24]: What do you think about combining these two into one line?

File: src/ui/components/Events/Events.tsx [hunk 3]
```diff
@@ -30,20 +45,18 @@ function Events({ currentTime, events, executionPoint, seek }: PropsFromRedux) {
...
   return (
-
-    <div className="flex flex-col py-1.5 self-stretch space-y-1.5 w-full text-xs bg-white">
-    
+    <div className="flex flex-col py-1.5 self-stretch w-full text-xs bg-white">
       {events.map((e, i) => {
         return (
           <div key={e.point}>
-            {i === currentEventIndex ? <CurrentTimeLine /> : null}
+            <CurrentTimeLine currentEventIndex={currentEventIndex} index={i} />
```
Comment (jcmorrow) [inline src/ui/components/Events/Events.tsx:52]: I could go either way on this, but it seems like maybe a lot to pass around these two variables just to do an equality check on them later. What do you think about having an `active` state, and this component can render `<CurrentTimeLine active={i === currentEventIndex}/>`?

File: src/ui/components/Events/Events.tsx [hunk 4]
```diff
@@ -30,20 +45,18 @@ function Events({ currentTime, events, executionPoint, seek }: PropsFromRedux) {
   );
 
   return (
-
-    <div className="flex flex-col py-1.5 self-stretch space-y-1.5 w-full text-xs bg-white">
-    
+    <div className="flex flex-col py-1.5 self-stretch w-full text-xs bg-white">
```
Comment (jcmorrow) [inline src/ui/components/Events/Events.tsx:48]: I think the only classes that are doing anything here are: `py-1.5 text-xs bg-white`?

**Review by jcmorrow — COMMENTED** [review]
I have some suggestions, but I don't feel super strongly about them.

File: src/ui/components/Events/Events.tsx (hunk 3 above)
Comment (jaril) [reply src/ui/components/Events/Events.tsx:52]: 👍

File: src/ui/components/Events/Events.tsx (hunk 4 above)
Comment (jaril) [reply src/ui/components/Events/Events.tsx]: 👍

**Comment by jaril** [comment]
Went ahead and made the changes 👍

https://user-images.githubusercontent.com/15959269/141353874-72a58036-b6ac-42c2-be38-9f778cdbb373.mov

**Review by jasonLaster — APPROVED** [review]

**Pull request merged**